lib/Doctrine/ORM/EntityNotFoundException.php

```diff
@@ -48,4 +50,18 @@ public static function fromClassNameAndIdentifier($className, array $id)
             'Entity of type \'' . $className . '\'' . ($ids ? ' for IDs ' . implode(', ', $ids) : '') . ' was not found'
         );
     }
+
+
+    /**
+     * Instance where no identifier can be found
+     *
+     * @param string $className
```

Comment:
When using type declaration, this because unnecessary, let's change it for something that adds information.

```suggestion
     * @psalm-param class-string $className
```